crates/sui-framework/packages/sui-framework/tests/crypto/bls12381_tests.move

```diff
@@ -1,12 +1,24 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+#[allow(implicit_const_copy)]
 #[test_only]
 module sui::bls12381_tests {
     use sui::bls12381;
-    use std::vector;
+    use sui::group_ops;
     use std::hash::sha2_256;
-    
+    use std::vector;
+    use sui::test_utils::assert_eq;
+
+    const ORDER_BYTES: vector<u8> = x"73eda753299d7d483339d80809a1d80553bda402fffe5bfeffffffff00000001";
+    const ORDER_MINUS_ONE_BYTES: vector<u8> = x"73eda753299d7d483339d80809a1d80553bda402fffe5bfeffffffff00000000";
+    const LONG_SCALAR_BYTES: vector<u8> = x"73eda753299d7d483339d80809a1d80553bda402fffe5bfeffffffff0000000000";
+    const SHORT_SCALAR_BYTES: vector<u8> = x"73eda753299d7d483339d80809a1d80553bda402fffe5bfeffffffff0000";
```

Comment:
what is the purpose of SHORT_SCALAR_BYTES?

Reply:
we expect fixed size inputs for deserialization, this is an example of a shorter input that should fail